src/main/python/mlsearch/helper.py

```diff
@@ -0,0 +1,49 @@
+def is_valid_parameters(event, param_names):
+    """
+    Check whether the item in param_names exist in event dictionary.
+    
+    :param  event:          Lambda event object.
+    :param  param_names:    The list of the param names to be checked.
+    
+    :retrun:                True if exist else False
+    """
+    for param in param_names:
+        if not param in event:
+            return False
+    return True
+
+def response(message, status_code):
```

Comment:
I found out that we still need to support CORS. I'll let you know the detail later in the meeting. 😄

Reply:
I fixed it at the lambda script. I guess it is an optional header and better to keep it out from ml_search api requester.